.github/workflows/advance_upstream_forks.yml

```diff
@@ -35,3 +35,25 @@ jobs:
           github_token: ${{ secrets.WRITE_ACCESS_TOKEN }}
           branch: main
           repository: google/iree-llvm-fork
+
+  advance_iree-mhlo-fork:
+    name: "Advance iree-mhlo-fork"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          repository: google/iree-mhlo-fork
+          ref: main
+
+      - name: Pull upstream main
+        run: |
+          git remote add upstream https://github.com/tensorflow/mlir-hlo.git
+          git pull --ff-only upstream master
```

Comment:
should these branch names match? One is `main` while the other is `master`

Reply:
Yes, thank you. For both repos they match the name of the branch upstream